test/Microsoft.DotNet.Tools.Tests.Utilities/NonWindowsOnlyFactAttribute.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.DotNet.PlatformAbstractions;
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Test.Utilities
+{
+    public class NonWindowsOnlyFactAttribute : FactAttribute
+    {
+        public NonWindowsOnlyFactAttribute()
+        {
+            if (RuntimeEnvironment.OperatingSystemPlatform == Platform.Windows)
+            {
+                this.Skip = "This test requires windows to run";
```

Comment:
Is this message correct? If it's a **non**-Windows only, I would expect it to say *"This test requires non-windows platform to run"*.